contents/handbook/engineering/incidents.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Handling an incident
+sidebar: Handbook
+showTitle: true
+---
+
+Incidents are going to happen.
+
+## When to raise an incident
+
+**When in doubt, raise an incident.** We'd much rather have declared an incident which turned out not to be an incident. Many incidents take too long to get called, or are missed completely because someone didn't ring the alarm when they had a suspicion something was wrong.
+
+To declare an incident, type `/incident` in the #incidents channel. This will create a new channel and send updates.
+
+Anyone can declare an incident, including non-engineers. If in doubt, check with your nearest engineer.
+
+Some things that should definitely be an incident
+- `app.posthog.com` being completely unavailable (not just for you)
+- No insights can be created
+- Feature flags are not being returned at all, or `/decide` is down
+- Various alerts defined as critical, such as disk space full, OOM or >5 minute ingestion lag
+
+Things that _shouldn’t_ be an incident
+- Insights returning incorrect data
+- Events being < 5 minutes behind
+- Unable to save insights, create feature flags
+
+## What happens during an incident
+
+The person who raised the incident is the incident lead. It’s their responsibility to:
+- Make sure the right people join the zoom. This includes the current on call person (https://posthog.pagerduty.com/service-directory/P43Y0E8), people from Infra and the feature owner if relevant (https://posthog.com/handbook/engineering/feature-ownership)
```

Comment:
Is the expectation that infra folks will join _every_ incident? If yes, why?

Reply:
Genuine question, what kind of incidents do you think would not benefit/be resolved quicker if infra folks aren't there?

Reply:
Any incident where whoever gets paged can quickly identify the issue and follow a runbook to fix it. 

Example:
- page for elevated overall error rate
- oncall person declares an incident 
- dashboard shows the error rate is coming from a specific endpoint
- oncall person identifies a PR related to that endpoint was recently deployed
- oncall person reverts the PR
- error rate goes back to nominal values 
- incident is declared mitigated
- once all follow up tasks are completed (eg notified the person/team responsible for the endpoint) the responder calls the "all clear"

Reply:
Make sense! I've made it "Optional if relevant" for both feature owner and platform folks.